src/LSST/M1M3/SS/FPGA/SimulatedFPGA.cpp

```diff
@@ -118,8 +152,15 @@ void SimulatedFPGA::pullTelemetry() {
     this->supportFPGAData.InclinometerErrorTimestamp = 0;
     this->supportFPGAData.InclinometerErrorCode = 0;
     this->supportFPGAData.InclinometerSampleTimestamp = timestamp;
-    this->supportFPGAData.InclinometerAngleRaw =
-            (int32_t)(tmaElevation.Elevation_Angle_Actual * 1000.0) + (this->getRnd() * 5.0);
+
+    // Inclinometer raw value is measured as zenith angle. Converts elevation to zenith angle.
```

Comment:
Please  change that to:

// Inclinometer raw value is measured as (negative) zenith angle (0 = zenith, -90 = horizon). Converts elevation to zenith angle and adds random 5 deg noise